### PR TITLE
Fix task table locator

### PR DESCRIPTION
**Pull request by swadeley**
Hello

This change works in Firefox Dev Edition's console. But still not able to read the table in a test.

### CODE REVIEW

File: airgun/views/task.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class TasksView(BaseLoggedInView, SearchableViewMixin):
         "//div[./button[@id='tasks-dashboard-time-period-dropdown']]"
     )
     table = SatTable(
-        ".//div[@id='tasks-table']/table",
+        ".//div[@class='tasks-table']",
```
Comment (mirekdlugosz) [inline airgun/views/task.py]: I'm pretty sure `Table` widget should match `<table>` element, so you want to retain this `/table` part.

Also, seems in latest snap entire table is wrapped in `<div>`? I think that wasn't there before and looks like a small bug. So make that `//table`.

Comment (swadeley) [reply airgun/views/task.py]: OK, // table works, /table did not.
The extra <div> is there in previous snap too.
Thank you

Comment (swadeley) [reply airgun/views/task.py]: @mirekdlugosz thanks a lot, that fixed the reading of the table in my testing.

**Comment by swadeley** [comment]
Before I got empty dict, like so []

No it works:

In [2]: session.task.search('Update package for g5vni1949pbkhpui.lab.eng.rdu2.redhat.com walrus')                 
Out[2]: 
[{'Select all rows': 'Select row',
  'Action': 'Update package for g5vni1949pbkhpui.lab.eng.rdu2.redhat.com walrus',
  'State': 'stopped',
  'Result': 'warning',
  'Started at': 'March 25, 2020, 11:41:27 AM',
  'Duration': '21 seconds',
  'Operation': 'Cancel'}]

In [3]:

**Comment by swadeley** [comment]
thank you all

**Pull request merged**